### PR TITLE
usage overview docu

### PR DESCRIPTION
**Pull request by ghedamat**
I'm not entirely sure what you guys envisioned to be in this page

I took a stab at describing the commands for now

happy to get feedback as usual

/cc @achambers @lukemelia

### CODE REVIEW

File: docs/v0.5.x/usage-overview.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+**Example:**
+
+```
+ember deploy production
+```
+
+### `ember deploy:activate`
+
+Runs the `activate*` hooks that are normally configured by your index plugin and are used to select a `revision` as the primary one to be served by your backend.
```
Comment (lukemelia) [inline docs/v0.5.x/usage-overview.md]: There is no concept of an "index plugin" any more. Perhaps  "Runs a pipeline with the `activate*` hooks that some plugins implement to make a specific `revision` as the active one to be served by your backend."

**Comment by lukemelia** [comment]
I think this is pretty good overall. Is there anything else you think is necessary @achambers?

File: docs/v0.5.x/usage-overview.md (hunk 1 above)
Comment (ghedamat) [inline docs/v0.5.x/usage-overview.md]: right, I guess I still reason in that mentality :D

**Comment by ghedamat** [comment]
@lukemelia should be golden

**Pull request merged**